### DISCUSSION TITLE
How to do gradient clipping in tsai?

### DISCUSSION THREAD

**Discussion by marcuswang6** [timeseriesAI/tsai#522] [Q&A] [answered]
How to do gradient clipping in tsai?

**Comment by marcuswang6** [comment, marked as answer]
Can I use the callback in fastai like ` GradientClip(), MixedPrecision()`?

**Reply by oguiza** [reply]
Yes, they should work the same way

**Reply by marcuswang6** [reply]
> Yes, they should work the same way

Thanks for your reply, oguiza!
